### DISCUSSION TITLE
Not using timestamp to sign packages with Hero

### DISCUSSION THREAD

**Discussion by ajpow9598** [marcinotorowski/MSIX-Hero#151] [Q&A]
We are re-signing packages due to a change in our code-signing certificate.  I would like to automat using the CLI.  However, we are not using a timestamp.  The command to sign and leaving off the --TimeStamp parameter is not working.  It appears to keep looking for a URL.  Can this be avoided?

Error:  Signing failed: SignTool.exe returned with exit code 0x01. The specified timestamp server either could not be reached or

Command:  msixherocli.exe sign --file <cert.pfx> --password <pwd> "<package>.msix"

**Comment by ajpow9598** [comment]
msixherocli.exe sign --file <cert.pfx> --password  x   "xxx.msix"

**Comment by cocoon** [comment]
should work with this change:
https://github.com/marcinotorowski/MSIX-Hero/pull/158
